tests/test_speedify.py

```diff
@@ -53,20 +206,18 @@ def test_connect_country(self):
 
     def test_transport(self):
         mysettings = speedify.transport("https")
-        serverinfo = speedify.connect()
```

Comment:
Why add the argument here? Did the behavior of `connect` change? It should be valid to call `connect` with no arguments and have it use the current connectmethod.